src/TricksBundle/Controller/TricksController.php

```diff
@@ -50,7 +48,7 @@ public function viewAction(Request $request, Trick $trick, $commentsPage = 1, Co
 
         // Paginated comments
         $nbCommentsPerPage = 10; // For Comments pagination
-        $trickComments = $this->em->getRepository('STTricksBundle:TrickComment')->findAllByTrickPaginated($trick, $commentsPage, $nbCommentsPerPage);
```

Comment:
For easy refactoring and better IDE support, you can replace this "magic" string by the complete class name getRepository(TrickComment::class) instead of getRepository('STTricksBundle:TrickComment')

Reply:
Thanks for the trick !